backend/Modules/Identity/HitMeApp.Indentity/Core/Policies/IPasswordStrengthPolicy.cs

```diff
@@ -0,0 +1,9 @@
+﻿using HitMeApp.Shared.DDD;
+
+namespace HitMeApp.Indentity.Core.Policies
+{
+    internal interface IPasswordStrengthPolicy : IPolicy
+    {
+        public void Test(string password);
```

Comment:
Shouldn't this return `bool` indicating whether password meets criteria? Or even better - some kind of enum indicating password overall strength tested against multiple factors (length, special characters, etc.)
```csharp
enum PasswordStrenght
{
   VeryWeak = 0,
   Weak,
   Normal,
   Strong,
   VeryStrong,
}
```

Reply:
Hmm. Seems like a cool idea! Let's talk it through on a call.